### PR TITLE
Error when description is less than 10 chars

### PR DESCRIPTION
**Pull request by jetersen**
### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### Description
Give Warning when Beta Description is less than 10 characters or greater than 4000

### Motivation and Context
Bumper to get denied after the upload is successful.
If I could be pointed in the right direction for a test case of pilot update build info

```
[16:32:36]: Successfully finished processing the build
[16:32:36]: You can now tweet: 
[16:32:36]: iTunes Connect #iosprocessingtime 10 minutes
*snip lane context*
[16:32:37]: [English]: Beta App Description must be at least 10 characters long. Beta App Description must be at least 10 characters long.
```

### CODE REVIEW

File: pilot/lib/pilot/build_manager.rb [hunk 1]
```diff
@@ -67,6 +67,7 @@ def distribute(options, build = nil)
 
       unless config[:update_build_info_on_upload]
         if should_update_build_information(options)
+          UI.user_error!("Beta App Description must be at least 10 characters long.") if options[:beta_app_description].to_s.length < 10 && options[:beta_app_description].to_s.length > 0
```
Comment (yukiasai) [inline pilot/lib/pilot/build_manager.rb]: There are other lines called `update_build_information!` method. If it fails in all cases, I think we should validate with `verify_block` in `options.rb`.

Comment (casz) [inline pilot/lib/pilot/build_manager.rb]: Sure thing, I am still new to ruby 👍

Comment (yukiasai) [inline pilot/lib/pilot/build_manager.rb]: @casz No probs. It's nice PR😊

File: pilot/lib/pilot/options.rb [hunk 2]
```diff
@@ -46,7 +46,10 @@ def self.available_options
                                      short_option: "-d",
                                      optional: true,
                                      env_name: "PILOT_BETA_APP_DESCRIPTION",
-                                     description: "Provide the beta app description when uploading a new build"),
+                                     description: "Provide the beta app description when uploading a new build",
+                                     verify_block: proc do |value|
+                                       UI.user_error!("Beta App Description must be at least 10 characters long.") unless options[:beta_app_description].to_s.length >= 10
```
Comment (ohayon) [inline pilot/lib/pilot/options.rb]: Can you check the `value.to_s.length` rather than `options[:beta_app_description]`?

**Review by ohayon — CHANGES_REQUESTED** [review]
This is a good change, just one little comment above. 🐙

File: pilot/lib/pilot/options.rb (hunk 2 above)
Comment (casz) [inline pilot/lib/pilot/options.rb]: Copy paste error, my bad!

**Comment by casz** [comment]
Added both minimum and maximum requirement 👍

**Review by KrauseFx — CHANGES_REQUESTED** [review]
We try to avoid having user input checks from within _fastlane_ that actually happen on iTunes Connect or the Developer Portal, as those requirements might change in the future. Many of the user checks on Apple's side can even be changed on a developer-account level, meaning there are different requirement/numbers per account. 

Thanks for the contribution, but we'll not go ahead with this. Instead, to make the error message more clear, I'd be all for showing the iTunes Connect error message in a nicer way, if it's not the case yet 👍

**Pull request closed without merging**